src/kbmod/work_unit.py

```diff
@@ -280,7 +285,11 @@ def from_fits(cls, filename):
             per_image_wcs = []
             per_image_ebd_wcs = []
             constituent_images = []
-            for i in range(n_constituents):
+            for i in tqdm(
```

Comment:
I'm assuming the process that takes long is the one that creates the `LayeredImage` (above). Is there a need for feedback on this as well?